src/sentry/models/artifactbundle.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    region_silo_only_model,
+)
+
+
+@region_silo_only_model
+class ArtifactBundle(Model):
+    __include_in_export__ = False
+
+    bundle_id = models.CharField(max_length=32, null=True)
+    organization_id = BoundedBigIntegerField(db_index=True)
+    # TODO: define max length.
+    release_name = models.CharField(null=True)
+    dist_id = BoundedBigIntegerField(null=True)
+    file = FlexibleForeignKey("sentry.File")
+    artifact_count = BoundedPositiveIntegerField()
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_artifactbundlefile"
+
+        unique_together = (("organization_id", "bundle_id"),)
+        index_together = (("organization_id", "release_name"),)
+
+
+@region_silo_only_model
+class DebugIdIndex(Model):
+    __include_in_export__ = False
+
+    debug_id = models.CharField(max_length=32, unique=True, db_index=True)
+    artifact_bundle = FlexibleForeignKey("sentry.ArtifactBundle")
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_debugidindex"
+
+
+@region_silo_only_model
+class ProjectArtifactBundleIndex(Model):
```

Comment:
nit: I would drop the `Index` part of these intermediate tables, see for example

https://github.com/getsentry/sentry/blob/7c58e7cb0ec0b20052dabbeb243ac0e896985b96/src/sentry/models/projectteam.py#L43-L47

Reply:
I have followed the `FileBlobIndex` pattern but I agree, it's not an index. It's its own table.